hack/generated/pkg/genruntime/arm_transformer.go

```diff
@@ -10,9 +10,8 @@ import (
 )
 
 type ToArmConverter interface {
-	// name is the "a/b/c" name for the owner. For example this would be "VNet1" when deploying subnet1
-	// or myaccount when creating Batch pool1
-	ConvertToArm(name string) (interface{}, error)
+	// ConvertToArm converts this to an ARM resource. The name parameter is the fully qualified name of the resource in Azure ("a/b/c").
+	ConvertToArm(name string) (interface{}, error) // TODO: Should be ConvertToARM (same below)
```

Comment:
Does this TODO need to be actioned before you merge?

Reply:
No, I've got a follow-on PR that does a bunch of `Arm` -> `ARM` renames but I didn't want to mix real changes and renames. Once this is merged I'll send the capitalization fixups for existing methods.

Reply:
Good plan.